modules/virt-change-vm-instance-type-cli.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vm/virt-creating-vms-from-instance-types.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-change-vm-instance-type-cli_{context}"]
+= Changing the VM instance type by using the CLI
+
+[role="_abstract"]
+As a VM owner, if you want to change the instance type of a VM, you can do this by changing the `name` field in the VM spec. This triggers the update logic, ensuring a new, immutable controller revision snapshot is taken of the new resource configuration.
+
+.Prerequisites
+
+* You created the VM by using an instance type.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Stop the VM.
+
+. Run the following command, replacing `<vm_name>` with the name of your VM, and `<new_instancetype>` with the name of the new instance type you want to change to:
++
+[source,terminal]
+----
+$ oc patch vm/<vm_name> --type merge -p '{"spec":{"instancetype":{"name": "<new_instancetype>"}}}'
+----
+
+.Verification
+
+* Check the controller revision reference in the updated VM `status` field by running the following command and verifying that the revision name is updated in the output:
++
+[source,terminal]
+----
+$ oc get vms/<vm_name> -o json | jq .status.instancetypeRef
+----
++
+.Example output
+[source,terminal]
+----
+{
+  "controllerRevisionRef": {
+    "name": "vm-cirros-csmall-csmall-3e86e367-9cd7-4426-9507-b14c27a08671-2"
+  },
+  "kind": "VirtualMachineInstancetype",
+  "name": "csmall"
+}
+----
+
+* You can also check that the VM instance is running the new configuration defined in the latest controller revision. For example, if you updated the instance type to use 2 vCPUs instead of 1, you could run the following command and check the output:
```

Comment:
```suggestion
. Optional: Check that the VM instance is running the new configuration defined in the latest controller revision. For example, if you updated the instance type to use 2 vCPUs instead of 1, run the following command and check the output:
```